frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarOverviewTab.tsx

```diff
@@ -0,0 +1,93 @@
+
+import { playerMetaLogic } from '../playerMetaLogic'
+import { sessionRecordingPlayerLogic } from '../sessionRecordingPlayerLogic'
+
+interface OverviewItem {
+    label: string
+    value: ReactNode
+    tooltipTitle?: string
+}
+
+const browserPropertyKeys = ['$geoip_country_code', '$browser', '$device_type', '$os']
+const mobilePropertyKeys = ['$geoip_country_code', '$device_type', '$os_name']
+const recordingPropertyKeys = ['click_count', 'keypress_count', 'console_error_count'] as const
+
+function sessionPlayerMetaDataToOverviewItems(sessionPlayerMetaData: SessionRecordingType | null): OverviewItem[] {
+    const items: OverviewItem[] = []
+
+    recordingPropertyKeys.forEach((property) => {
+        if (sessionPlayerMetaData?.[property]) {
+            items.push({
+                label: getCoreFilterDefinition(property, TaxonomicFilterGroupType.Replay)?.label ?? property,
+                value: sessionPlayerMetaData[property],
+            })
+        }
+    })
+
+    const personProperties = sessionPlayerMetaData?.person?.properties ?? {}
+
+    const deviceType = personProperties['$device_type'] || personProperties['$initial_device_type']
+    const deviceTypePropertyKeys = deviceType === 'Mobile' ? mobilePropertyKeys : browserPropertyKeys
+
+    deviceTypePropertyKeys.forEach((property) => {
+        if (personProperties[property]) {
+            const value = personProperties[property]
+
+            const tooltipTitle =
+                property === '$geoip_country_code' && value in countryCodeToName
+                    ? countryCodeToName[value as keyof typeof countryCodeToName]
+                    : value
+
+            items.push({
+                label: getCoreFilterDefinition(property, TaxonomicFilterGroupType.PersonProperties)?.label ?? property,
+                value: <PropertyIcon property={property} value={value} />,
+                tooltipTitle,
+            })
+        }
+    })
+
+    return items
+}
+
+function OverviewCard({ item }: { item: OverviewItem }): JSX.Element {
+    return (
+        <Tooltip title={item.tooltipTitle}>
+            <div className="flex-1 p-2 text-center">
+                <div className="text-sm">{item.label}</div>
+                <div className="text-lg font-semibold">{item.value}</div>
+            </div>
+        </Tooltip>
+    )
+}
+
+export function OverviewCardRow({ items }: { items: OverviewItem[] }): JSX.Element {
+    return (
+        <div className="grid grid-cols-3 place-items-center rounded border bg-bg-light m-2">
+            {items.map((item) => (
+                <OverviewCard key={item.label} item={item} />
+            ))}
+        </div>
+    )
+}
+
+export function PlayerSidebarOverviewTab(): JSX.Element {
+    const { logicProps } = useValues(sessionRecordingPlayerLogic)
+    const { sessionPlayerMetaData } = useValues(playerMetaLogic(logicProps))
+
+    const items = sessionPlayerMetaDataToOverviewItems(sessionPlayerMetaData)
```

Comment:
would probably have this as a selector on the playerMetaLogic

(only fly-by haven't looked at this PR other than to scan it :))

Reply:
Done!